tests/test_simulation.py

```diff
@@ -3825,6 +3816,12 @@ def test_averaging_per_body_nodal(
         ref_data = get_ref_per_body_results_mechanical(ref_files[result], mesh)
 
     def get_expected_label_space_by_mat_id(mat_id: int):
+        # mapdl_element_type_id is not part of the label space before DPF 9.1
```

Comment:
@PProfizi Agreed, I was not aware this functionality has recently been added (https://tfs.ansys.com:8443/tfs/ANSYS_Development/_git/DPF/commit/128ccd3b1b666d72f9fe9b014ca40d6c25b03237?_a=compare&path=%2FAns.Dpf.Native%2FScopingOperators.cpp)